### ISSUE TITLE
Transaction name isn't logged

### ISSUE THREAD

**Issue by jameswyse** [winterbe/mobx-logger#6]
I'm getting `%s` in my console instead of the transaction name:

```
21:31:08.725        TX  %s
21:31:08.732    ACTION  GridStore.setOptions()
21:31:08.739    ACTION  GridStore.setData()
21:31:08.754        TX  %s
```

Looks like the name wasn't passed to `groupCollapsed` in [log.js - line 31](https://github.com/winterbe/mobx-logger/blob/master/src/log.js#L31) - adding it as the last arg seems to fix it 👍 

```js
console.groupCollapsed('%c%s  %s  %s', style('#7B7B7B'), now(), padStart('TX', 8), ev.name);
```

**Comment by winterbe** [comment]
Thanks for reporting this. Please keep in mind that Transactions are deprecated in Mobx 3 so you better migrate to `runInAction`: 

https://github.com/mobxjs/mobx/blob/master/CHANGELOG.md#other-changes

**Comment by winterbe** [comment]
I've fixed this for upcoming version 0.4.0 which requires Mobx 3 as peer dependency.
